server/index.js

```diff
@@ -1,19 +1,20 @@
 const express = require('express');
-const app = express();
 const url = require('url');
-let port = 3000;
-let models = require('./models/index.js');
+const path = require('path');
 
-app.use(express.urlencoded());
-app.use(express.static(__dirname + '../client'));
+const app = express();
 
+const port = 3000;
+const models = require('./models/index.js');
 
+app.use(express.urlencoded());
+app.use(express.static(path.join(__dirname, '../client')));
 
-app.get('/api/images', (req, res) => {
+app.get('/api/:hostel_id/images', (req, res) => {
```

Comment:
This looks like a lot better endpoint for ease of routing than your previous endpoint; it will be much easier to plop the hostel_id into your db query without needing to parse anything.